extras/update_hashes.py

```diff
@@ -0,0 +1,15 @@
+import glob
+import os, os.path as osp
```

Comment:
I'd rather see this as

```py
import os
import os.path
```

But if you'd rather use `osp` then put it on its own line I'd say. I'm guessing `os.path` makes some of these lines more than 80 characters but since we aren't PEP8ing I'd rather see a longer line than a non-standard usage of `os.path`.